plugin/src/main/java/org/openrewrite/gradle/RewritePlugin.java

```diff
@@ -70,8 +73,38 @@ public void apply(Project project) {
 
         // Rewrite module dependencies put here will be available to all rewrite tasks
         Configuration rewriteConf = project.getConfigurations().maybeCreate("rewrite");
+        rewriteConf.getDependencies().addAllLater(
```

Comment:
Have you also tried the `testGradle4` task as well? I think `addAllLater` was added in a newer version, so this may break the lower bound of version compatibility.

Reply:
Running the CI task now as well, so we'll know from that as well soon.

Reply:
Didn't know about that task.. and it looks like you're right. Let me fix that.

Reply:
Looks like this is the failing test, which I believe is one of the two that are failing on the mainline. I could fix the other one, but I don't know what's going on with this one:  
```
RewriteRunTest > lombokTypeAttribution(File) FAILED
    org.opentest4j.AssertionFailedError: 
    expected: 
      "package org.openrewrite.before;
  
      import lombok.Getter;
  
      public class N {
          @Getter
          int n;
  
          public static void foo() {
              System.out.println(/*~~>*/new N().getN());
          }
      }"
     but was: 
      "package org.openrewrite.before;
  
      import lombok.Getter;
  
      public class N {
          @Getter
          int n;
  
          public static void foo() {
              System.out.println(new N().getN());
          }
      }"
        at app//org.openrewrite.gradle.RewriteRunTest.lombokTypeAttribution(RewriteRunTest.kt:1564)
 ```